### DISCUSSION TITLE
ColorScheme

### DISCUSSION THREAD

**Discussion by VikaramjitSinghGorraya** [chakra-ui/chakra-ui#2961] [Q&A]
What is the difference between using colorScheme and color?

**Comment by chr-ge** [comment]
- `color` is used to give text a color 
- `colorScheme` prop represents a visual color scheme, not a css color attribute. 
Read more about it here: https://github.com/chakra-ui/chakra-ui/discussions/2846#discussioncomment-228303, https://github.com/chakra-ui/chakra-ui/discussions/2846#discussioncomment-230531

**Comment by pom421** [comment]
For what I understand, using colorScheme is a way to not have to specify all color stuff for your components. 

For Button, for example, using colorScheme="blue", will use blue.500 for the default state and blue.600 for the :hover state.

**Comment by rezamajidi** [comment]
`colorScheme` has a whole set of visual color attributes of `background-color` and `color` for _normal_ and _hover_ state with it but, color is just a `textColor` for an attribute.

Try adding a `colorScheme="green"` to a button and check the result. now remove the `colorScheme` and just add a `textColor` or `bg`.

**Comment by renemichel** [comment]
Is there a way to add a custom `colorScheme` ?

**Reply by felipeduardo** [reply]
@renemichel yes, https://chakra-ui.com/docs/styled-system/customize-theme#using-theme-extensions you can use this tool to create a color pallete https://themera.vercel.app/
